test/functional/Jsr292/modules/mods.modulec/module-info.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017, 2018 IBM Corp. and others
+ * Copyright (c) 2017, 2020 IBM Corp. and others
```

Comment:
copyright only change?  Moving a file doesn't change it's contents and shouldn't update the copyright date

Reply:
Restoring the copyright back to the original date for these moved files led to errors in the copyright check as follows at https://ci.eclipse.org/openj9/job/PullRequest-CopyrightCheck-OpenJ9/23580/console:
```
11:51:50  ###################################
[Pipeline] echo
11:51:50  The following files were modified and have incorrect copyrights
[Pipeline] echo
11:51:50  test/functional/Jsr292/modules/mods.modulea/module-info.java
[Pipeline] echo
11:51:50  test/functional/Jsr292/modules/mods.moduleb/module-info.java
[Pipeline] echo
11:51:50  test/functional/Jsr292/modules/mods.modulec/module-info.java
[Pipeline] echo
11:51:50  test/functional/Jsr292/src/com/ibm/j9/jsr292/CustomClassLoader.java
```

Reply:
The copyright checker is wrong here - just add a comment to the PR that says why the check is failing is sufficient

Reply:
Comment was added above at https://github.com/eclipse/openj9/pull/8943#issuecomment-602148401